apple/MarkdownTextFieldObserver.h

```diff
@@ -0,0 +1,19 @@
+#import <UIKit/UIKit.h>
+#import <React/RCTUITextField.h>
+#import <RNLiveMarkdown/RCTMarkdownUtils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MarkdownTextFieldObserver : NSObject
+
+@property (nonatomic, nullable, strong) RCTMarkdownUtils *markdownUtils;
+
+@property (nonatomic, nullable, strong) RCTUITextField *textField;
+
+@property (nonatomic) BOOL active;
```

Comment:
This looks like it doesn't need to be public.

Reply:
Removed in 40abeba